data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -0,0 +1,141 @@
+Jupyter Notebooks can be identified by their suffix ".ipynb". 
+
+**Using a Terminal:**
+The DP0.1 data sets and the LSST Science Pipelines tools can both be accessed from the command line of a terminal. As described in the default message that appears in all newly-launched terminals, to create a Rubin Observatory environment in a terminal session and set up the full set of packages, users must first execute "source ${LOADSTACK}" and "setup lsst_distrib". For example, to query and retrieve DP0.1 data sets using the Butler (see below), command-line tools are available as documented `here <https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/scripts/butler.html>`_. Type "butler --help" in any terminal to see a list of available butler functionality.
+
+**Safely Log Out of JupyterLab:**
+Use the File item in the top menu bar.
+To safely shut down a Notebook, choose Close and Shutdown Notebook.
+To safely shut down a JupyterLab server and log out of the RSP, choose Save all, Exit, and Log Out.
+
+
+.. _NB-Intro-Use-A-NB:
+
+How to Use a Jupyter Notebook
+=============================
+
+The best way to learn how to use a Jupyter Notebook is to open the first of the tutorial notebooks which are provided in all users' home directories, and also available in the `tutorial-notebooks <https://github.com/rubin-dp0/tutorial-notebooks>`_ repository in the ``rubin-dp0`` GitHub Organization (see also :ref:`Examples-DP0-1-Notebooks`). 
+
+**Executing code in a Notebook:**
+Jupyter notebooks provide "cells" within which you type either Python code or markdown language (for formatted text).
+Choose the cell to execute by clicking in it with your mouse (the cursor must be in the desired cell).
+Hold down the `shift` key and press either `enter` or `return` (depending on your keyboard type), and the contents of the cell will be executed.
+If the cell type is code, and the cell contains python code, the code will be compiled and executed.
+If the cell type is markdown, then it will be rendered upon execution to yield nicely formatted text.
+(For some handy markdown tips, see `this blog post <https://medium.com/analytics-vidhya/the-ultimate-markdown-guide-for-jupyter-notebook-d5e5abf728fd>`_ or the `relevant section from the JupyterLab documentation <https://jupyter-notebook.readthedocs.io/en/latest/examples/Notebook/Working%20With%20Markdown%20Cells.html#Markdown-Cells>`_). 
+
+.. figure:: /_static/notebook.png
+    :name: notebook_aspect
+
+    An example of what users see in the Notebook Aspect. The left panel is a file tree from which you can access your personal files, and which will come pre-populated with a handful of tutorial notebooks that users can execute on their own. The largest panel is the Jupyter notebook, where markdown and code can be entered and executed. You can also see that one can have multiple notebooks open in separate tabs.
+
```

Comment:
Another kind of general comment, is that you mention that there can be multiple notebooks in separate tabs, but an even more powerful feature that was a game changer is that you can actually tile the various notebooks and terminals next to each other or on top of each other so you can see multiple notebooks at once.

This is very powerful if you are e.g. trying to follow along with another notebook, but maybe only use part of it for your own purpose.

Obviously you don't want to be rewriting the JupyterLab user guide, but I use the feature virtually every time I log into the RSP.